examples/all-clusters-app/linux/main-common.cpp

```diff
@@ -46,6 +46,8 @@
 #include <transport/SessionManager.h>
 #include <transport/raw/PeerAddress.h>
 
+#include <app/icd/IcdManagementServer.h> // nogncheck
```

Comment:
I don't think the `// nogncheck` is necessary here.

Reply:
yes it is, otherwise it wouldn't be there.

Reply:
Why tho? It is not conditionally included, so this is not a case of gn check not doing the preprocessor defines.  So what is it nocheck solving?